domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowConditionParser.java

```diff
@@ -1,28 +1,70 @@
 package uk.gov.hmcts.ccd.definition.store.domain.showcondition;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
+import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 @Component
 public class ShowConditionParser {
 
+    private static final Logger LOG = LoggerFactory.getLogger(ShowConditionParser.class);
+
+    private static final String AND_CONDITION_REGEX = "\\sAND\\s(?=(([^\"]*\"){2})*[^\"]*$)";
+    private static final String AND_OPERATOR = " AND ";
+    private static final Pattern EQUALITY_CONDITION_PATTERN = Pattern.compile("\\s*?(.*)\\s*?(=)\\s*?(\".*\")\\s*?");
+
     public ShowCondition parseShowCondition(String rawShowConditionString) throws InvalidShowConditionException {
         try {
-            Matcher matcher = Pattern.compile("\\s*?(.*)\\s*?(=)\\s*?(\".*\")\\s*?").matcher(rawShowConditionString);
-            if (matcher.find()) {
-                return new ShowCondition.Builder()
-                                .showConditionExpression(matcher.group(1).trim() + matcher.group(2).trim() + matcher.group(3).trim())
-                                .field(matcher.group(1).trim())
-                                .build();
-
+            String[] andConditions = rawShowConditionString.split(AND_CONDITION_REGEX);
+            Optional<ShowCondition> optShowCondition = buildShowCondition(andConditions);
+            if (optShowCondition.isPresent()) {
+                return optShowCondition.get();
             }
-        }
-        catch (Exception e) {
+        } catch (Exception e) {
+            LOG.error("Error occurred while parsing show condition", e);
```

Comment:
@gbenadikar shall we log this as a WARN? seems a bit too much to me logging as ERROR. But I'm not 100% sure. you chose :)

Reply:
@mario-paniccia there is an exception thrown outside the catch block which stops the import, hence I chose error rather than warn.